overlord/snapstate/handlers.go

```diff
@@ -2283,14 +2283,14 @@ func (m *SnapManager) doLinkSnap(t *state.Task, _ *tomb.Tomb) (err error) {
 		return err
 	}
 
-	// Prepare for rebooting for snaps that need it. Note that if we have
+	// Set next boot for snaps that need it. Note that if we have
 	// kernel-modules components this gets delayed as it happens in the
 	// "prepare-kernel-modules-components" task. The default is set to
 	// true for compatibility with older snapd (case of joint refresh of
 	// snapd and kernel).
 	var rebootInfo boot.RebootInfo
 	prepareRestart := true
```

Comment:
shouldn't we change the var name as well? same in other places?

Reply:
Done